packages/coreutils/src/statedb.ts

```diff
@@ -255,13 +255,18 @@ export namespace StateDB {
     /**
      * The contents of the change operation.
      */
-    contents: ReadonlyJSONObject | null;
+    contents: Content<T> | null;
   };
 
+  /**
+   * Database content map
```

Comment:
Should this be plural instead since we call it `contents` in `DataTransform`?

Also, the doc string should end in a full stop even when it's not a sentence 😆

Reply:
Do you mean just in the docstring, or the type name as well (`Contents`)?

Reply:
I mean `StateDB.Contents`.

Reply:
Hm, perhaps you're correct. I don't know which is best.

From: https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents

> Content (uncountable)
> 
> Content as a singular noun is uncountable. It means the ideas that are contained in a piece of writing or a film or a speech:
> 
>>   This film has adult content; it is not suitable for children.
> 
>>    His speech was very flowery, but it lacked content and was quite superficial.
> 
> Content can also mean the amount of a particular substance contained in something:
> 
>>    These hamburgers have a high fat content; they’re not good for you.
> 
> Contents (plural)
> 
> The contents of a book is the list of chapters or articles or parts that are in the book, with the number of the page they begin on:
> 
>>    If you look at the contents, you’ll see there’s a chapter on Japanese folk music.

Reply:
If you are happy with 'Content', then this might be ready to merge. If you still think 'Contents', then I can make the change later. I'll leave the decision up to the English-as-a-first-language crowd.